src/test/java/seedu/address/logic/commands/AddPersonalTaskCommandTest.java

```diff
@@ -114,9 +114,11 @@ private AddPersonalTaskCommand getAddPersonalTaskCommandForTask(PersonalTask tas
     private class ModelStubThrowingTimingClashException extends ModelStub {
         @Override
         public void addTask(Task task) throws TimingClashException {
-            throw new TimingClashException(MESSAGE_TASK_TIMING_CLASHES);
```

Comment:
since the message will be passed to user through command exception, the message is not really useful.

Reply:
then at least the message should be at AddPersonalTaskCommand.java instead of having it in TimingClashException.java [Same for AddTuitionTaskCommand.java]